src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -187,11 +192,15 @@ def eq_distillate_temp(b):
                 == b.feed_props[0].temperature + b.delta_T_last_effect
             )
 
+        # Salinity in distillate is zero
+        @self.Constraint(doc="distillate salinity")
+        def eq_distillate_salinity(b):
+            return b.distillate_props[0].flow_mass_phase_comp["Liq", "TDS"] == 0
```

Comment:
You could also fix to zero and skip constraint, but if tests are passing, it might be OK for now.

Reply:
Yes it makes sense. This constraint is removed.